src/profile-logic/symbolication.js

```diff
@@ -143,7 +145,12 @@ import type {
  * This is different to initial symbolication, which only ever needs to *merge*
  * funcs, not split them, because the initial profile starts out in a
  * "maximally-split" state: every frame has its own function at the beginning of
- * initial symbolication.
+ * initial symbolication. (This is true except in one case: Some frames were
```

Comment:
I'll try to rephrase it. "This" is referring to the statement 'the initial profile starts out in a "maximally-split" state'. After the return address nudging, the profile may not be maximally split any more, because we may have some duplicated frames which use the same func.

Reply:
I'm not sure I understand it all, but that's fine 😁